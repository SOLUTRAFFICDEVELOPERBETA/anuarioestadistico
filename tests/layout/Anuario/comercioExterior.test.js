import React from 'react';
import '@testing-library/jest-dom';
import { mount, shallow } from 'enzyme';
import ComercioExterior from '../../../Layout/Anuario/comercioExterior';
import PagesContext from '../../../contexts/pagess';
import { pagesTest } from '../../fixtures';


describe('Pruebas en ComercioExterior', () => {

    const wrapper = mount(
        <PagesContext.Provider value={{
            pages: pagesTest
        }}>
            <ComercioExterior />
        </PagesContext.Provider>
    )
    test('debe de mostrar el componente correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe retornar la informaciÃ³n del modulo filtrado, dependiendo del tipo de campo', () => {
        expect(wrapper.find('h5').text().trim()).toBe('Valor de prueba')
    })

})