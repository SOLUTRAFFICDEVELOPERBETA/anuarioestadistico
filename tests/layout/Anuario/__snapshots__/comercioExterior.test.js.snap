// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en ComercioExterior debe de mostrar el componente correctamente 1`] = `
<div>
  <div>
    <div
      className="MuiBox-root MuiBox-root-1"
    >
      <div
        className="MuiBox-root MuiBox-root-2 css-771n28-SectionGeneric"
      >
        <div
          className="css-1x4qkb1-SectionGeneric"
        >
          <h5
            className="MuiTypography-root css-y9wiai MuiTypography-h5 MuiTypography-colorPrimary MuiTypography-alignCenter"
            style={
              Object {
                "fontWeight": 700,
                "paddingBottom": "1rem",
              }
            }
          >
            Valor de prueba
          </h5>
        </div>
      </div>
    </div>
  </div>
</div>
`;
